src/SymmetricIntervalHull.jl

```diff
@@ -28,13 +28,13 @@ bounds in the `cache` field.
 """
 struct SymmetricIntervalHull{N<:Real, S<:LazySet{N}} <: AbstractHyperrectangle{N}
     X::S
-    cache::Vector{Tuple{N, Bool}}
+    cache::Vector{N}
 
     function SymmetricIntervalHull{N, S}(X::S) where {S<:LazySet{N}} where {N<:Real}
-        cache = Vector{Tuple{N, Bool}}(dim(X))
-        zero_N = zero(N)
+        cache = Vector{N}(dim(X))
+        minus_one_N = -one(N)
         for i in 1:length(cache)
-            cache[i] = (zero_N, false)
+            cache[i] = minus_one_N
```

Comment:
(the same but shorter: `fill!(cache, -one(N)`, or even `cache = fill(-one(N), dim(X))`.)